openerp-client/src/main/java/org/bahmni/openerp/web/http/client/RestClient.java

```diff
@@ -79,6 +78,28 @@ public String post(String URL, String requestBody) {
         }
     }
 
+    private HashMap<String, String> getSession(ResponseCookie cookie) {
+        HashMap<String, String> session = new HashMap<>();
+        session.put("session_id", cookie.getValue());
```

Comment:
I think instead of converting the cookie object into a map and adding custom validation, it is better to store the cookie as `ResponseCookie` object itself and use the methods provided by the library for validations.